src/ja/user/message/query.py

```diff
@@ -52,7 +52,7 @@ def label(self) -> List[str]:
         return self._label
 
     @property
-    def owner(self) -> List[str]:
+    def owner(self) -> List[int]:
```

Comment:
The owners should be a list of strings, not a list of ints.
This is the thing that the user types in (usernames).
At some point on the server these usernames can be translated into Linux user IDs.